script.transmission/default.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
this file should be as small as possible, only used to import your main codebase, propagate `sys.argv` and run. Reason is this is the only file that is not compiled to bytecode by kodi's python interpreter.

You should consider do something like:

```python
from sys import argv
from resources.lib import myscript

myscript.run(argv)
```